mongo/client.go

```diff
@@ -728,9 +751,17 @@ func (c *Client) UseSessionWithOptions(ctx context.Context, opts *options.Sessio
 	return fn(sessCtx)
 }
 
-// Watch returns a change stream cursor used to receive information of changes to the client. This method is preferred
+// Watch returns a change stream used to receive information of changes to the client. This method is preferred
```

Comment:
How about: "Watch returns a change stream for all data changes on the connected deployment".  I would not mention "preferred to $changeStream" as that's extra information that might be confusing and I suspect few people would do it anyway.

Reply:
I'd prefer not to use "data changes" because change streams will return events for database/collection drops, which are arguably not "data" changes. I changed it to match your suggestion but used "changes" instead of "data changes" and added a link to the official change stream docs.

I initially thought the line about "preferred to $changeStream" was a spec documentation requirement but I don't see it in the spec so I removed it.